.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -0,0 +1,67 @@
+# Manual Maintenance Sprint [NR]
```

Comment:
Mit dem File wird der Abschnitt in der Readme obsolet - wir sollten das nicht 2x pflegen.

Reply:
ok